i18n/oceans.json

```diff
@@ -63,8 +63,7 @@
     "run": "Run",
     "mostImportantParts": "These were the most important fish parts:",
     "clickIndividualFish": "Click individual fish to see their information.",
-    "mostImportantPartsInDetermining": "These were the most important fish parts in determining whether this fish was",
-    "or": "or",
+    "mostImportantPartsInDeterminingFullSentence": "These were the most important fish parts in determining whether this fish was “*{word}*” or “**{notWord}**”.",
```

Comment:
`..."FullSentence"` seems a bit awkward.  How about just `"mostImportantParts"`?

Reply:
`mostImportantParts` already exists as a key :)

We also can't reuse the old keys (or else I would've used `mostImportantPartsInDetermining` again). The key isn't actually that important and I can't really think of anything better but definitely open to suggestions!

Reply:
`mostImportantPartsDescription`?